lib/src/async_cache.dart

```diff
@@ -44,14 +53,18 @@ class AsyncCache<T> {
   /// The [duration] starts counting after the Future returned by [fetch]
   /// completes, or after the Stream returned by `fetchStream` emits a done
   /// event.
-  AsyncCache(Duration duration) : _duration = duration;
+  AsyncCache(Duration duration, {bool cacheErrors = true})
```

Comment:
```suggestion
  /// If [cacheErrors] is `false` the cache will be invalidated if the [Future]
  /// returned by the callback completes as an error.
  AsyncCache(Duration duration, {bool cacheErrors = true})
```

Reply:
We have given usage of `cacheErrors` above where `_cacheErrors`  is declared